tools/klee-stats/klee-stats

```diff
@@ -21,30 +21,47 @@ import collections
 # Mapping of: (column head, explanation, internal klee name)
 # column head must start with a capital letter
 Legend = [
+    # core stats
     ('Instrs', 'number of executed instructions', "Instructions"),
-    ('Time(s)', 'total wall time (s)', "WallTime"),
-    ('TUser(s)', 'total user time', "UserTime"),
+    ('Time(s)', 'total wall time', "WallTime"),
     ('ICov(%)', 'instruction coverage in the LLVM bitcode (%)', "ICov"),
-    ('BCov(%)', 'branch coverage in the LLVM bitcode (%)', "BCov"),
+    ('BCov(%)', 'branch (br) coverage in the LLVM bitcode (%)', "BCov"),
     ('ICount', 'total static instructions in the LLVM bitcode', "ICount"),
-    ('TSolver(s)', 'time spent in the constraint solver', "SolverTime"),
-    ('TSolver(%)', 'time spent in the constraint solver', "RelSolverTime"),
-    ('States', 'number of currently active states', "NumStates"),
-    ('MaxStates', 'maximum number of active states', "maxStates"),
-    ('AvgStates', 'average number of active states', "avgStates"),
-    ('Mem(MB)', 'megabytes of memory currently used', "MallocUsage"),
-    ('MaxMem(MB)', 'megabytes of memory currently used', "MaxMem"),
-    ('AvgMem(MB)', 'megabytes of memory currently used', "AvgMem"),
-    ('Queries', 'number of queries issued to STP', "NumQueries"),
-    ('AvgQC', 'average number of query constructs per query', "AvgQC"),
-    ('Tcex(s)', 'time spent in the counterexample caching code', "CexCacheTime"),
-    ('Tcex(%)', 'relative time spent in the counterexample caching code wrt wall time', "RelCexCacheTime"),
-    ('Tfork(s)', 'time spent forking', "ForkTime"),
-    ('Tfork(%)', 'relative time spent forking wrt wall time', "RelForkTime"),
+    ('TSolver(%)', 'time spent in the solver chain (incl. caches and constraint solver) wrt wall time', "RelSolverTime"),
+    # extended stats
+    # - code and coverage
+    ('ICovered', 'total covered instructions in the LLVM bitcode', "CoveredInstructions"),
+    ('IUncovered', 'total uncovered instructions in the LLVM bitcode', "UncoveredInstructions"),
+    ('Branches', 'number of conditional branch (br) instructions in the LLVM bitcode', 'NumBranches'),
+    ('FullBranches', 'number of fully-explored branch (br) instructions in the LLVM bitcode', 'FullBranches'),
+    ('PartialBranches', 'number of partially-explored conditional branch (br) instructions in the LLVM bitcode', 'PartialBranches'),
+    # - time
+    ('TUser(s)', 'total user time', "UserTime"),
     ('TResolve(s)', 'time spent in object resolution', "ResolveTime"),
     ('TResolve(%)', 'time spent in object resolution wrt wall time', "RelResolveTime"),
+    ('TCex(s)', 'time spent in the counterexample caching code', "CexCacheTime"),
```

Comment:
It's Cex and solver, isn't it?

Reply:
Oh dear... you're right, that was fixed in MoKlee I think.